README.md

```diff
@@ -16,14 +16,13 @@ Start the replica, then build and install the canisters.
 
 ```bash
 dfx start --background
+dfx canister create --all
 dfx build
 dfx canister install --all
 ```
 
 Open the canister frontend in your web browser.
 
 ```bash
-ID=$(xxd -u -p canisters/linkedup/_canister.id)
-CRC=$(python2 -c "import crc8;h=crc8.crc8();h.update('$ID'.decode('hex'));print(h.hexdigest().upper())")
-xdg-open "http://127.0.0.1:8000/?canisterId=ic:$ID$CRC"
+xdg-open "http://127.0.0.1:8000/?canisterId=$(dfx canister id linkedup_assets)"
```

Comment:
is `xdg-open` available on mac? `open` exhibits the same behavior for me

Reply:
It wasn't available for me either